Config/tasmanianConfig.hpp

```diff
@@ -27,13 +27,27 @@
  * THE USER ASSUMES RESPONSIBILITY FOR ALL LIABILITIES, PENALTIES, FINES, CLAIMS, CAUSES OF ACTION, AND COSTS AND EXPENSES, CAUSED BY, RESULTING FROM OR ARISING OUT OF,
  * IN WHOLE OR IN PART THE USE, STORAGE OR DISPOSAL OF THE SOFTWARE.
  */
- 
-#ifndef __TASMANIAN_SPARSE_GRID_VERSION_HPP
-#define __TASMANIAN_SPARSE_GRID_VERSION_HPP
+
+#ifndef __TASMANIAN_CONFIG_HPP
+#define __TASMANIAN_CONFIG_HPP
 
 #define TASMANIAN_VERSION_MAJOR 5
 #define TASMANIAN_VERSION_MINOR 1
 #define TASMANIAN_VERSION_STRING "5.1"
 #define TASMANIAN_LICENSE "BSD 3-Clause with UT-Battelle disclamer"
```

Comment:
The simple make command uses this file and technically it is generated on configuration time. I edit one .in.hpp file, then cmake creates two copies, one to be used with the current cmake build with the selected options and one with all options disabled so it can be used with simple make.